### PR TITLE
[gatsby-remark-images] Fixed issue with wrapperStyle not being applied correctly

### PR DESCRIPTION
**Pull request by iamhitarth**
Fix for #7201 .

`wrapperStyle` option was not being applied correctly when `showCaptions` was set to `true` so this PR fixes that.

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit f2a12490ba2856e7724023f3a31694e68dd119dd

https://app.netlify.com/sites/using-postcss-sass/deploys/5b72aea8c6aed63fd12b6312

**Comment by KyleAMathews** [comment]
Deploy preview for *using-contentful* failed.

Built with commit f2a12490ba2856e7724023f3a31694e68dd119dd

https://app.netlify.com/sites/using-contentful/deploys/5b72aeaac6aed63fd12b6321

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbyjs* failed.

Built with commit f2a12490ba2856e7724023f3a31694e68dd119dd

https://app.netlify.com/sites/gatsbyjs/deploys/5b72aea7c6aed63fd12b6306

File: packages/gatsby-remark-images/src/index.js [hunk 1]
```diff
@@ -172,7 +172,7 @@ module.exports = (
   <span
     class="gatsby-resp-image-wrapper"
     style="position: relative; display: block; ${
-      options.wrapperStyle
+      options.showCaptions && node.title ? null : options.wrapperStyle
```
Comment (pieh) [inline packages/gatsby-remark-images/src/index.js]: Nitpicky - can we do `const showCaption = options.showCaptions && node.title` before this and then use that both here and in L204 to not multiplicate same logic to multiple places?

Other than that, it looks great!

**Comment by pieh** [comment]
Published in `gatsby-remark-images@2.0.1-beta.11`

**Pull request merged**